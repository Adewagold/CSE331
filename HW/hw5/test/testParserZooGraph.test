# Test the parser on zoos' data to see 
# if the graph is built correctly

# Parse through the data and build the graph
LoadGraph zoos zoos.tsv

# List nodes and edges to see if the graph 
# is built correctly
ListNodes zoos
ListChildren zoos Monkey
ListChildren zoos Penguin
ListChildren zoos Koala
ListChildren zoos Elephant
ListChildren zoos Tiger