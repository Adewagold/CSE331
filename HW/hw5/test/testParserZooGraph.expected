# Test the parser on zoos' data to see 
# if the graph is built correctly

# Parse through the data and build the graph
loaded graph zoos

# List nodes and edges to see if the graph 
# is built correctly
zoos contains: Elephant Koala Monkey Penguin Tiger
the children of Monkey in zoos are: Elephant(Zoo2) Koala(Zoo1) Penguin(Zoo2) Tiger(Zoo1)
the children of Penguin in zoos are: Elephant(Zoo2) Monkey(Zoo2)
the children of Koala in zoos are: Elephant(Zoo3) Monkey(Zoo1) Tiger(Zoo1)
the children of Elephant in zoos are: Koala(Zoo3) Monkey(Zoo2) Penguin(Zoo2)
the children of Tiger in zoos are: Koala(Zoo1) Monkey(Zoo1)